lib/router/transition.js

```diff
@@ -16,7 +16,7 @@ import { trigger, slice, log, promiseLabel } from './utils';
   @param {Object} error
   @private
  */
-function Transition(router, intent, state, error) {
+function Transition(router, intent, state, error, previousTransition) {
```

Comment:
I tend to agree with @alexspeller here, but might make sense for @stefanpenner / @krisselden  / other @tildeio/ember-core folks to sanity check also...